storybooknative/src/Checkbox/index.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from "react";
+import { View } from "react-native";
+
+import styled from "../styled";
+import Touchable from "../Touchable";
+
+import Props from "./index";
+
+export default function Checkbox(props: Props) {
+  const StyledIconPlaceholder = styled(View, () => ({}));
+
+  const StyledWrapper = styled(View, () => ({}));
```

Comment:
Why are this styled wrapper without any styles used instead of just a `View`?

Reply:
Also if the intent is to add styles here later, the definition should still be outside of the component. Now it is being recreated on each render.

Reply:
Fixed the definition scope and yes, it is just a set up for next iteration where I am adding the styles.